ui/pages/permissions-connect/snaps/snap-install/index.scss

```diff
@@ -2,15 +2,19 @@
   box-shadow: none;
 
   &__content {
-    flex: 1;
-
     &__loader-container {
       height: 100%;
     }
   }
 
+  &__scroll-button {
+    position: absolute;
+    right: 160px;
```

Comment:
There can be use cases when this button will not be centered because pixels are used. Position of it can be affected depending on screen configuration, scrollbar presence, resolution scaling or pixel density. Also, if somebody extends popup (which is possible), it will be 160px from the right, which is not centered. Also, if at any time width of the extension changes, this will not be centered.

So, I would suggest updating this to something more flexible like:
```css
position: absolute;
bottom: 90px;
margin-left: auto;
margin-right: auto;
right: 0;
left: 0;
```
Not sure if it's the best solution so far, but it should handle most of the cases.
Also, this applies to the update flow as well.

Reply:
Good point - applied 👍